File: src/codingTestReady/백준4375번1.java
package codingTestReady;

import java.io.*;
import java.util.Scanner;

public class 백준4375번1 {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        while(sc.hasNextLong()){
            int n = sc.nextInt();
            int count = 1;
            long now = 1L;
            while(true){
                if(now % n == 0L) break;
                now = now*10 + 1;
                count += 1;
            }
            System.out.println(count);
        }
    }

}

File: src/codingTestReady/백준4375번1.kt
package codingTestReady

import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val bw = BufferedWriter(OutputStreamWriter(System.out))


    while(true){
        val line = br.readLine() ?: break
        bw.append(findSmallest(br.readLine().toInt()).toString()+"\n")
    }
    bw.flush()
    bw.close()

}

fun findSmallest(n: Int): Int {
    var count = 1
    var now = 1L
    while(true){
        if(now % n.toLong() == 0L) break
        now = now*10 + 1
        count += 1
    }
    return count
}
